sdk/options.go

```diff
@@ -99,18 +100,21 @@ func withCustomAccessTokenSource(a auth.AccessTokenSource) Option {
 	}
 }
 
+// Deprecated: Use WithCustomCoreConnection instead
 func WithCustomPolicyConnection(conn *grpc.ClientConn) Option {
```

Comment:
Should we just reroute this to the customcoreconnection?

Reply:
I am thinking about that. Does that get confusing and will that break anyone currently using multiple custom connections ?

Reply:
Yes, but if they are deprecated then the IDE should complain. A deployment may fail, but we can tell them that we no longer support that case. I'll leave it up to you.

Reply:
Updated